armory/datasets/README.md

```diff
@@ -0,0 +1,115 @@
+# Setup
+
+This is designed to be run in an armory pytorch container.
+Recommended use:
+```
+armory launch pytorch
```

Comment:
Should we add `build` and `load` as entry points in `pyproject.toml`? This way you can run this without installing armory; e.g. something like `docker run --rm -it --command armory-datasets build -v mount_volumes:if_needed twosixlabs/pytorch:latest`.

Reply:
Possibly, but let's hold off for now. We will want to think through most of the use cases beforehand. For most users, I don't think it will be needed, as loading a cached dataset within a scenario should work perfectly fine without its own entry point.

Reply:
Makes sense, just note we can have multiple entrypoints to different modules; `armory`, `armory-engine`, `armory-datasets`, etc.